scripts/src/main/resources/scripts/command/tomcat

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+# autocompletion list
+if [ "${1}" = "shortlist" ]
+then
+  if [ -z "${2}" ]
+  then
+    echo "setup version help"
+  fi
+  exit
+fi
+
+
+
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
+export CATALINA_HOME="${DEVON_IDE_HOME}/software/tomcat"
+TOMCAT_HOME="${DEVON_IDE_HOME}/software/tomcat"
+TOMCAT="${TOMCAT_HOME}/bin/startup.bat"
+TOMCAT="catalina.sh"
+if doIsWindows
+then
+  TOMCAT="catalina.bat"
+fi
```

Comment:
According to my tests, this can be omitted to simplify the code as `catalina.sh` works fine on windows as well.